courses/free-courses-fr.md

```diff
@@ -25,6 +26,13 @@
 * [Cours d'Algorithmie](https://www.youtube.com/playlist?list=PLrSOXFDHBtfGy7xYmf5LlNr8f-niDkf7_) - Formation Video
 
 
+### APL
+
+* [Découvrez le langage APL : 1er partie avec fonctions de base et exercices corrigés](https://www.youtube.com/watch?v=Nxq1BUUXobM) - Eric Schrafstetter
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes  Is the a playlist that includes all three?